actions/bump-version/action.yaml

```
@@ -33,12 +33,18 @@ runs:
     - name: Bump version
       id: bump-version
       run: |
-        parameters=(--no-commit --no-tag ${{ inputs.release-type }})
+        parameters=(--no-commit --allow-dirty --no-tag ${{ inputs.release-type }})
```

Since we basically want to use the first 3 parameters for all of our bumpversion CLI calls, what do you think about removing `${{ inputs.release-type }}` from the list and adding that separately, e.g.:
```suggestion
        parameters=(--no-commit --allow-dirty --no-tag)
```
And then for every call below:
```bash
bump2version "${parameters[@]}" major/minor/patch/release
```

---

Shouldn't this be used in basically all bump-my-version commands now?